src/services/user.ts

```diff
@@ -0,0 +1,6 @@
+import usersFromServer from '../api/users';
+import { User } from '../types/User';
+
+export function getUserById(userId: number): User | null {
```

Comment:
If `getUserById` is intended to be asynchronous (e.g., fetching from a server), consider returning a promise and using `async/await` in its implementation. Currently, it is synchronous, which is fine if `usersFromServer` is a static list.